### PR TITLE
added missing outputs to get-user command - Gmail

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20781

## Description
Added 3 missing outputs to get-user command

## Does it break backward compatibility?
   - No

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)

### CODE REVIEW

**Comment by Shellyber** [comment]
@kirbles19 Please review YML file, The `gmail-get-user` command.
Added 3 outputs: 
* Account.Group
* Account.CustomerId
* Accound.VisibleInDirectory

File: Integrations/Gmail/Gmail.yml [hunk 1]
```diff
@@ -1836,7 +1846,6 @@ script:
   longRunningPort: false
   runonce: false
   script: '-'
-  subtype: python2
```
Comment (Itay4) [inline Integrations/Gmail/Gmail.yml]: revert this

**Comment by Itay4** [comment]
add empty changelog

**Comment by kirbles19** [comment]
@Shellyber Done. When you create the changelog, it should be.

Added the following outputs to the ***gmail-get-user*** command.
  - *Account.Group*
  - *Account.VisibleInDirectory*
  - *Account.FustomerId*

**Pull request merged**